srcpkgs/nvidia-shortlived/template

```diff
@@ -0,0 +1,323 @@
+# Template file for 'nvidia-shortlived'
+
+_desc="NVIDIA drivers for linux (short lived series)"
+
+pkgname=nvidia-shortlived
+version=415.27
+revision=1
+maintainer="Colin Gillespie <colin@breavyn.com>"
+license="Proprietary NVIDIA license"
+homepage="http://www.nvidia.com"
+
+archs="i686 x86_64"
+nopie=yes
+repository="nonfree"
+create_wrksrc=yes
+short_desc="${_desc} - Libraries and Utilities"
+conflicts="catalyst>=0 xserver-abi-video>24_1 nvidia"
+
+if [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
+	depends="nvidia-shortlived-libs-${version}_${revision} nvidia-shortlived-gtklibs-${version}_${revision} nvidia-shortlived-dkms-${version}_${revision} pkg-config"
```

Comment:
cut this at 80 columns into a newline and prefix with a whitespace, same level as depends=

Reply:
The whole clause can go as it only supports one architecture anyways.